### PR TITLE
fix(docs): several typos

### PR DESCRIPTION
**Pull request by Pysics**
This PR fixes several typos.

### CODE REVIEW

File: source/plugins/base/metadata.yml [hunk 1]
```diff
@@ -21,10 +21,10 @@ inputs:
       * `header`, which usually contains username, two-week commits calendars and a few additional data
       * `activity`, which contains recent activity (commits, pull requests, issues, etc.)
       * `community`, which contains community stats (following, sponsors, organizations, etc.)
-      * `repositories`, which contains repositories stats (license, forks, stars, etc.)
-      * `metadata`, which contains informations about generated metrics
+      * `repositories`, which contains repository stats (license, forks, stars, etc.)
```
Comment (spenserblack) [inline source/plugins/base/metadata.yml:24]: Alternatively this could be `repositories' stats`, to mean "stats of multiple repositories"

Comment (Pysics) [reply source/plugins/base/metadata.yml:24]: I feel `repository stats` sounds better.

Comment (spenserblack) [reply source/plugins/base/metadata.yml:24]: I actually agree, just making sure the alternatives are out there for the maintainer to review.

Comment (Pysics) [reply source/plugins/base/metadata.yml:24]: :+1:

**Review by lowlighter — APPROVED** [review]

**Pull request merged**